pages/spicedb/modeling/migrating-schema.mdx

```diff
@@ -1,29 +1,28 @@
 import { Callout } from "nextra/components";
 
-# Updating and Migrating Schema in SpiceDB
+# Migrating a Schema in SpiceDB
 
 <Callout type="info">
-  This page explains how to modify a SpiceDB schema. If you need information
+  This page explains how to migrate a SpiceDB schema. If you need information
   about migrating the schema of a datastore underlying SpiceDB, like Postgres or
-  CockroachDB, go [here](/spicedb/concepts/datastore-migrations#migrations). If
+  CockroachDB, go [here](/spicedb/concepts/datastore-migrations). If
   you need information about migrating between SpiceDB instances, go
   [here](/spicedb/ops/data/migrations).
 </Callout>
 
-[Schema] in SpiceDB represents the structural definitions of which relationships are allowed
-in SpiceDB and how permissions are computed.
+[Schema] in SpiceDB is the file that represents the structural definitions of which relationships are allowed
+in SpiceDB and how permissions are computed. This file can be updated or migrated via the [WriteSchema] API.
```

Comment:
This reads a little weird to me... the thing being updated in SpiceDB isn't really a file.

Maybe instead of "file" we say "object in the database?"

Reply:
I took the easy way out on this one